about-pses/example-pse.md

```diff
@@ -58,20 +130,24 @@ Try not to pose questions that are too specific to the implementation. For examp
 
 As stated, this list contains several questions that overlap in what they are asking as an example of different perspectives. Try not to include multiple questions about the same topic in your own questions. For example, if we ask about how to handle invalid input, we only need to ask once, not in three different ways as we have here, especially since the sample answers here are contradictory!
```

Comment:
```suggestion
As stated, this list contains several questions that overlap in what they are asking as an example of different perspectives. *Try not to include multiple questions about the same topic in your own questions.*
- For example, if we ask about how to handle invalid input, we only need to ask once, not in three different ways as we have here, especially since the sample answers here are contradictory!
```